tests/testthat/test-shinytest2-module_bookmark_manager.R

```diff
@@ -3,7 +3,7 @@ testthat::test_that("bookmark_manager_button is not rendered by default", {
   app <- TealAppDriver$new(
     data = simple_teal_data(),
     modules = example_module(label = "Example Module"),
-    options = options()
+    options = list()
```

Comment:
@vedhav could you explain this change?

Reply:
It's cause the machine running the test can have different options which will make the tests perform differently.
For example in my machine by default, I set the testing depth to 5 in my global .Rprofile.
Sorry, I should have commented about this.

Reply:
No worries. All clear